packages/storybook/src/clinical/organisms/plan-definition-editor/plan-definition-editor.stories.tsx

```diff
@@ -0,0 +1,46 @@
+import { useState } from 'react'
+import { Story } from '@storybook/react'
+import Card from '@ltht-react/card'
+import PlanDefinitionEditor, { OnProblemChange } from '@ltht-react/plan-definition-editor'
+
+import { planDefinition } from './plan-definition-editor.fixtures'
+
+export const Editor: Story = () => {
+  const [selectedProblemIds, setSelectedProblemIds] = useState<string[]>([])
+
+  const handleProblemChange: OnProblemChange = (e) => {
+    // eslint-disable-next-line no-console
+    console.log('onProblemChange', e)

```

Comment:
I know it's only a story so really doesn't matter, but was this left in by mistake? Or does it serve a demo purpose in the storybook?

Reply:
It was in there as demo, but if it's unclear then I can remove it

Reply:
Nah all good, just checking :) Is it something that someone browsing the stories would need to see? 
If so I wonder if it's worth adding a note telling them to open devtools, or if perhaps we can surface the logs in a text container to the side?